roles/configure-nginx/files/debian-comp.conf

```diff
@@ -0,0 +1,2 @@
+# To be compatible with debian
+include /etc/nginx/sites-enabled/*.conf;
```

Comment:
Centos does not have sites-enabled sites-availlable.
My goal is to create thoses directories to made centos compatible with debian.